hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -445,6 +454,32 @@ message GetContainerTokenResponseProto {
   required TokenProto token = 1;
 }
 
+message StartContainerBalancerRequestProto {
+  optional string traceID = 1;
+  required double threshold = 2;
```

Comment:
Can we using KeyValue here if there will be more fields in future.

Reply:
Do all these required fileds have default values? If so, they are optional, insteal of required.

Reply:
Thanks @ChenSammi for the reivew. there will not be default value, see [this discuss](https://github.com/apache/ozone/pull/2278#discussion_r640330813)